apps/frontend/src/components/global/upload_tabs/DatabaseReader.vue

```diff
@@ -93,30 +88,18 @@ export default class DatabaseReader extends mixins(ServerMixin) {
     return EvaluationModule.allEvaluations;
   }
 
-  load_results(evaluations: IEvaluation[]): void {
-    Promise.all(
-      evaluations.map(async (evaluation) => {
-        return axios
-          .get<IEvaluation>(`/evaluations/${evaluation.id}`)
-          .then(({data}) => {
-            return InspecIntakeModule.loadText({
-              text: JSON.stringify(data.data),
-              filename: evaluation.filename,
-              database_id: evaluation.id,
-              createdAt: evaluation.createdAt,
-              updatedAt: evaluation.updatedAt,
-              tags: [] // Tags are not yet implemented, so for now the value is passed in empty
-            }).catch((err) => {
-              SnackbarModule.failure(err);
-            });
-          })
-          .catch((err) => {
-            SnackbarModule.failure(err);
-          });
-      })
-    ).then((fileIds: (FileID | void)[]) => {
-      this.$emit('got-files', fileIds.filter(Boolean));
+  joinEvaluationsByIds(evaluations: IEvaluation[] | Sample[]): string {
+    let stringresult = '';
+    evaluations.forEach((evaluation: IEvaluation | Sample) => {
+      if(evaluation.hasOwnProperty('id')){
+        stringresult += `${(evaluation as IEvaluation).id},`;
+      }
     });
+    return stringresult.slice(0, -1);
+  }
+
+  load_results(evaluations: IEvaluation[]): void {
+    this.$router.push(`/results/${this.joinEvaluationsByIds(evaluations)}`);
   }
```

Comment:
If the files aren't loaded all at once then this is not working correctly. (watch URL bar)

![sharingURL](https://user-images.githubusercontent.com/2308869/111226199-27501080-85b7-11eb-9708-92b14c23de61.gif)